### PR TITLE
Patch 4

### PR DESCRIPTION
**Pull request by Rashmi-278**
- Add  forum info and treasury address ; 
Radicle, rally, yearnfi, truefi

### CODE REVIEW

File: protocols/yearnfi/index.json [hunk 1]
```diff
@@ -3,5 +3,9 @@
 	"description": "",
 	"path": "ybaby.eth",
 	"isEnabled": true,
+	"discourseForum" : {
+		"url":"https://gov.yearn.finance/",
+		"categoryID":""
```
Comment (cryptosabago) [inline protocols/yearnfi/index.json:8]: curiosity qn: if they have no ID, should this still be filled in? If it should be filled in anyway, how does the dashboard interpret the empty string?

File: protocols/rally/index.json [hunk 2]
```diff
@@ -4,5 +4,9 @@
   "path": "rallygov.eth",
   "previousPaths": ["rally"],
   "isEnabled": true,
+  "discourseForum" : {
+    "url":"https://forum.rally.io/",
+    "categoryID":"15"
+  },
   "treasuryAddresses": ["0x3b66890289366175c360c1f0e1bafae06276ba03","0xAb9C84ad7f8B1356A370d367234fe215DF979Ec2"]
 }
```
Comment (cryptosabago) [inline protocols/rally/index.json:12]: nit: lint please

**Review by cryptosabago — APPROVED** [review]

File: protocols/rally/index.json (hunk 2 above)
Comment (Rashmi-278) [reply protocols/rally/index.json:12]: fixed

File: protocols/yearnfi/index.json (hunk 1 above)
Comment (Rashmi-278) [reply protocols/yearnfi/index.json]: Had missed it, fixed it.

**Review by CarlGocht — APPROVED** [review]

**Pull request merged**